tests/test_cli.py

```diff
@@ -1,12 +1,107 @@
-import os
+from pathlib import Path
 import pytest
-from junitparser.cli import verify
+from junitparser import cli
+from junitparser import version
+
+DATA_DIR = Path(__file__).parent / "data"
 
 
 @pytest.mark.parametrize(
     "file, expected_exitcode",
-    [("data/jenkins.xml", 1), ("data/no_fails.xml", 0), ("data/normal.xml", 1)],
+    [("jenkins.xml", 1), ("no_fails.xml", 0), ("normal.xml", 1)],
 )
-def test_verify(file, expected_exitcode):
-    path = os.path.join(os.path.dirname(__file__), file)
-    assert verify([path]) == expected_exitcode
+def test_verify(file: str, expected_exitcode: int):
+    path = DATA_DIR / file
+    assert cli.verify([path]) == expected_exitcode
+
+
+def test_merge():
+    files = [DATA_DIR / "jenkins.xml", DATA_DIR / "pytest_success.xml"]
+    suites = ["JUnitXmlReporter", "JUnitXmlReporter.constructor", "pytest"]
+    outfile = DATA_DIR / "merged.xml"
+    cli.merge(files, str(outfile))
+    xml = outfile.read_text()
```

Comment:
I'd prefer tests not to write into the project directory, better use a temporary file:
```suggestion
    with NamedTemporaryFile(suffix=".xml") as outfile:
      cli.merge(files, str(outfile.name))
      xml = outfile.read()
```
This needs an `from tempfile import NamedTemporaryFile`.
The `tests/data/.gitignore` is then not required.

Reply:
pretty sure pytest comes with a fixture for that, I will look into it later